File: ABC/abc015/abc015_2.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int N, A, s = 0, num = 0; // s: バグの合計数, num: バグがあるソフトウェアの個数
    cin >> N;
    for (int i = 0; i < N; i++) {
        cin >> A;
        if (A != 0) {
            s += A;
            num++;
        }
    }

    cout << (s + num - 1) / num << endl;
}

File: ABC/abc015/abc015_2.py
N = int(input())

s = 0 #バグの合計数
num = 0 #バグがあるソフトウェアの個数
for A in map(int, input().split()):
    if A != 0:
        s += A
        num += 1

print((s + num - 1) // num)